src/API/GitLab/GitLabAPI.php

```diff
@@ -48,6 +51,19 @@ protected function apiClient()
         );
     }
 
+    public static function determineGitLabUrl(Config $config)
+    {
+        // Start by seeing if GitLab CI has set a potential Project URL for us.
+        $potential_ci_url = getenv('CI_PROJECT_URL');
```

Comment:
If we use config, then it is not necessary to call `getenv` directly. The config item `build-tools.provider.git.gitlab.url` can already be overridden with the environment variable `TERMINUS_BUILD_TOOLS_PROVIDER_GIT_GITLAB_URL`.

Reply:
Ahh, I didn't realize it pulled from environment variables. As I was refactoring this I realized it would be just as simple to set our own environment variable based on the creator's config vs trying to parse a GitLab CI variable. With the naming convention in place, we can remove some of this code.